components/screens/MenuScreen.tsx

```diff
@@ -93,8 +96,22 @@ export const MenuScreen = (queryCache: QueryCache, avalancheCenterId: AvalancheC
   const {data} = useAvalancheCenterMetadata(avalancheCenterId);
   const menuItems = settingsMenuItems[avalancheCenterId];
 
+  const {
+    preferences: {mixpanelUserId},
+  } = usePreferences();
+  const [updateGroupId] = useState(getUpdateGroupId());
```

Comment:
We're not holding onto this state or mutating it in the future, so I don't think you need the `useState()` here:

```ts
const updateGroupId = getUpdateGroupId();
```

Reply:
total newb question, it seems like it wouldn't be needed here either then? https://github.com/NWACus/avy/pull/647/files#diff-93271258411bbda48ad01e2d70e98d5079555570679f1f32dad049f300874dd4L24